run/idp-sql/pom.xml

```diff
@@ -30,8 +30,8 @@ limitations under the License.
   </parent>
 
   <properties>
-    <maven.compiler.target>17</maven.compiler.target>
-    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
All code samples are supposed to run on Java 8. How do you plan to ensure that the code can run on Java 8?

Reply:
The original sample was written in Java 17. I only downgraded it due to prior ask. I have now downgraded it again.